builder/ebs/builder_acc_test.go

```diff
@@ -1231,6 +1323,61 @@ build {
 }
 `
 
+const testWindowsFastBoot = `
+source "amazon-ebs" "windows-fastboot" {
+	ami_name             = "%s"
+	source_ami           = "ami-00b2c40b15619f518" # Windows server 2016 base x86_64
+	instance_type        = "m3.medium"
+	region               = "us-east-1"
+	communicator         = "winrm"
+	winrm_username       = "Administrator"
+	winrm_password       = "e4sypa55!"
+	user_data_file       = "test-fixtures/ps_enable.ps"
+	fast_launch {
+		target_resource_count = 1
+	}
+}
+
+build {
+	sources = ["amazon-ebs.windows-fastboot"]
+
+	provisioner "powershell" {
+		inline = [
+			"C:/ProgramData/Amazon/EC2-Windows/Launch/Scripts/InitializeInstance.ps1 -Schedule",
+			"C:/ProgramData/Amazon/EC2-Windows/Launch/Scripts/SysprepInstance.ps1 -NoShutdown"
+		]
+	}
+}
+`
+
+const testWindowsFastBootWithTemplateID = `
+source "amazon-ebs" "windows-fastboot" {
+	ami_name             = "%s"
+	source_ami           = "ami-00b2c40b15619f518" # Windows server 2016 base x86_64
+	instance_type        = "m3.medium"
+	region               = "us-east-1"
+	communicator         = "winrm"
+	winrm_username       = "Administrator"
+	winrm_password       = "e4sypa55!"
+	user_data_file       = "test-fixtures/ps_enable.ps"
+	fast_launch {
+		target_resource_count   = 1
+		template_id = "lt-0c82d8943c032fc0b"
```

Comment:
Where does this template id come from?

Reply:
This is the default template created by AWS when requesting fast-launch to be enabled. Since we run those acceptance tests on our dev account, this is already created and we can reuse it.
Alternatively we could use the name: `EC2FastLaunchDefaultResourceCreation-30886dd2-77ed-4e79-b9d8-f543dfa1c800`